app/helper/rtForm.php

```diff
@@ -662,6 +722,31 @@ public function display_link( $args = '' ) {
 			echo $this->get_link( $args );
 		}
 
+		/**
+		 * Get rtmedia html input file in admin options.
+		 *
+		 * @access public
+		 *
+		 * @param string|array $attributes Arguments to create file input control.
+		 *
+		 * @return string
+		 */
+		public function get_inputfile( $attributes = '' ) {
+			return $this->generate_inputfile( $attributes );
+		}
+
+
+		/**
+		 * Display file input settings for set custom thumbnail generator.
+		 *
+		 * @param array $args Arguments to create file input control.
+		 *
+		 * @throws rtFormInvalidArgumentsException Invalid argument exception.
+		 */
+		public function display_inputfile( $args = '' ) {
```

Comment:
There is array in parameter description but empty string is assigned to `$args`. You can change parameter type to `array|string`.